src/registries/menus/items/format_menu_items.ts

```diff
@@ -0,0 +1,360 @@
+  name: _lt("Bold"),
+  description: "Ctrl+B",
+  action: ACTIONS.FORMAT_BOLD_ACTION,
+  icon: "o-spreadsheet-Icon.BOLD",
+  isSelected: (env) => !!env.model.getters.getCurrentStyle().bold,
+};
+
+export const formatItalicMenuItem: MenuItemSpec = {
+  name: _lt("Italic"),
+  description: "Ctrl+I",
+  action: ACTIONS.FORMAT_ITALIC_ACTION,
+  icon: "o-spreadsheet-Icon.ITALIC",
+  isSelected: (env) => !!env.model.getters.getCurrentStyle().italic,
+};
+
+export const formatUnderlineMenuItem: MenuItemSpec = {
+  name: _lt("Underline"),
+  description: "Ctrl+U",
+  action: ACTIONS.FORMAT_UNDERLINE_ACTION,
+  icon: "o-spreadsheet-Icon.UNDERLINE",
+  isSelected: (env) => !!env.model.getters.getCurrentStyle().underline,
+};
+
+export const formatStrikethroughMenuItem: MenuItemSpec = {
+  name: _lt("Strikethrough"),
+  action: ACTIONS.FORMAT_STRIKETHROUGH_ACTION,
+  icon: "o-spreadsheet-Icon.STRIKE",
+  isSelected: (env) => !!env.model.getters.getCurrentStyle().strikethrough,
+};
+
+export const formatFontSizeMenuItem: MenuItemSpec = {
+  name: _lt("Font size"),
+  children: fontSizeMenuBuilder(),
+};
+
+export const formatAlignmentMenuItem: MenuItemSpec = {
+  name: _lt("Alignment"),
+  icon: "o-spreadsheet-Icon.ALIGN_LEFT",
+};
+
+export const formatAlignmentHorizontalMenuItem: MenuItemSpec = {
+  name: _lt("Horizontal align"),
+  icon: "o-spreadsheet-Icon.ALIGN_LEFT",
+};
+
+export const formatAlignmentLeftMenuItem: MenuItemSpec = {
+  name: _lt("Left"),
+  action: (env: SpreadsheetChildEnv) => ACTIONS.setStyle(env, { align: "left" }),
+  isSelected: (env) => getHorizontalAlign(env) === "left",
+  icon: "o-spreadsheet-Icon.ALIGN_LEFT",
+};
+
+export const formatAlignmentCenterMenuItem: MenuItemSpec = {
+  name: _lt("Center"),
+  action: (env: SpreadsheetChildEnv) => ACTIONS.setStyle(env, { align: "center" }),
+  isSelected: (env) => getHorizontalAlign(env) === "center",
+  icon: "o-spreadsheet-Icon.ALIGN_CENTER",
+};
+
+export const formatAlignmentRightMenuItem: MenuItemSpec = {
+  name: _lt("Right"),
+  action: (env: SpreadsheetChildEnv) => ACTIONS.setStyle(env, { align: "right" }),
+  isSelected: (env) => getHorizontalAlign(env) === "right",
+  icon: "o-spreadsheet-Icon.ALIGN_RIGHT",
+};
+
+export const formatAlignmentVerticalMenuItem: MenuItemSpec = {
+  name: _lt("Vertical align"),
+  icon: "o-spreadsheet-Icon.ALIGN_MIDDLE",
+};
+
+export const formatAlignmentTopMenuItem: MenuItemSpec = {
+  name: _lt("Top"),
+  action: (env: SpreadsheetChildEnv) => ACTIONS.setStyle(env, { verticalAlign: "top" }),
+  isSelected: (env) => env.model.getters.getCurrentStyle().verticalAlign === "top",
+  icon: "o-spreadsheet-Icon.ALIGN_TOP",
+};
+
+export const formatAlignmentMiddleMenuItem: MenuItemSpec = {
+  name: _lt("Middle"),
+  action: (env: SpreadsheetChildEnv) => ACTIONS.setStyle(env, { verticalAlign: "middle" }),
+  isSelected: (env) => (env.model.getters.getCurrentStyle().verticalAlign || "middle") === "middle",
```

Comment:
We should have a constant `DEFAULT_VERTICAL_ALIGN` and use it here and in the renderer. Smae for the wrapping.